src/components/Heading.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledHeading = styled.div`
+  color: var(--tints-blizzard-black);
+  text-align: center;
+  padding: 4rem 1.5rem 3rem;
```

Comment:
According to the design file we are going to need to add and update some code for this. 

1. It looks like this element should be flex, column direction with gap of .5rem between children. 
2. Padding should be '0'. 
3. If we are going to keep 'max-width' property, we'll need to horizontally center it. 
4. Also 'width' for this element is '539px' which is about '33.688rem'.

Reply:
Thanks for spotting all this. Didn't address it all 1:1 but it should behave a bit better. 
- Set to display flex
- Kept padding so that it div doesn't touch side of viewport when less than max width
- horizontally centered the component
- Kept width to let content determine width